README.md

```diff
@@ -1,24 +1,165 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+<div align="center">
+  <h1>BUDGET APP</h1>
+  <br/>
 
-* Ruby version
+</div>
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* Database creation
 
-* Database initialization
+# 📖<a name="about-project">Budget App</a>
 
-* How to run the test suite
+> "Budget App"  is a mobile web application where users can manage your budget: you have a list of transactions associated with a category, so that users can see how much money they spent and on what.
 
-* Services (job queues, cache servers, search engines, etc.)
+<br/>
 
-* Deployment instructions
+# App  Screenshots <a name="screenshots"></a>
+  <img src="app/assets/images/homepage.png" alt="">
 
-* ...
+  <img src="app/assets/images/page1.png" alt="">
+
+  <img src="app/assets/images/page2.png" alt="">
+<details>
+  <summary>Tech stacks</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+
+  </ul>
+</details>
+<br>
+
+## Key Features <a name="key-features"></a>
+
+- **User can create an account..**
+- **Registered users will be able to create new category and operation..**
+- **Users will be able to view operations and category on the application's home page.**
+- **User can login..**
+- **Rspec test for models, controllers and views..**
+
+## 🚀 Project documentation 
+
+
+### Take a look at project's [Live Demo](https://budgetapp-7px9.onrender.com/)
```

Comment:
- [ ] Kindly fix the link to your live version, there's a black screen on that link